.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: PR Checks
+on:  
```

Comment:
There is an extra space at the end of line 3.

Reply:
Ohh my bad 🙈 . Let me remove it and raise it again